lib/core/treenode.js

```diff
@@ -105,6 +106,8 @@ class TreeNode {
 
     const {args, stackTrace, options} = this;
 
+    HttpRequest.globalSettings = Object.assign({}, HttpRequest.globalSettings, {redactParams: false});
```

Comment:
this shouldn't be here.

Reply:
So the problem with using 'globalSetting' of HttpRequest is that, after we have set `redactParams` to `true`, it remains 'true' in subsequent requests. So the future logs of other commands will also be modified, so I added it to prevent that.

Reply:
Cc @beatfactor Taggin again, as this conversation got lost 😛

Reply:
It didn't get lost. This change is inadequate and we need to find a better way of implementing this. I haven't had time to look into other options.

Reply:
I will revisit this after we have released alpha version